make/autoconf/jdk-version.m4

```diff
@@ -75,8 +75,6 @@ AC_DEFUN_ONCE([JDKVER_SETUP_JDK_VERSION_NUMBERS],
   . $TOPDIR/closed/autoconf/version-numbers
 
   # Some non-version number information is set in that file
-  AC_SUBST(LAUNCHER_NAME)
-  AC_SUBST(PRODUCT_NAME)
```

Comment:
This file doesn't need to change to support custom launcher and product names; the customizations belong in `custom-hook.m4`. Setting those variables even as late as in `CUSTOM_LATE_HOOK` is effective.

Reply:
I guess that can work for LAUNCHER_NAME, but $PRODUCT_NAMEis used in this file to set the JDK_RC_NAME and needs to have the updated value.

Reply:
As discussed, moving the customizations is fragile (it doesn't deal well with other references to `LAUNCHER_NAME` or `PRODUCT_NAME`).